src/System.Net.Primitives/tests/PalTests/System.Net.Primitives.Linux.Pal.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
> What still is unclear to me is why do these run only on certain OSs. Couldn't they be enabled for all OSs?

They could, as they do not call into platform-specific native code. I hadn't thought of that--I can file an issue for follow-up.